elastic/kibana
# [Maps] fix mapbox glyphs error when EMS access is turned off

Fixes https://github.com/elastic/kibana/issues/66799

Add rest endpoint to serve open sans fonts from Kibana server so fonts are available when EMS is turned off.

To test:
1) stop kibana server
2) set `map.includeElasticMapsService: false` in kibana.dev.yml
3) start kibana server
4) create new map and add cluster layer. Labels should render
